RELEASING.md

```diff
@@ -0,0 +1,41 @@
+# Releasing OpenTelemetry Packages (for Maintainers Only)
+
+This document explains how to publish all OT modules at version x.y.z. Ensure that you’re following semver when choosing a version number.
+
+## Use the Changelog to create a GitHub Release
+
+### Generate the changelog with lerna
+
+Since we use `lerna`, we can use [lerna-changelog](https://github.com/lerna/lerna-changelog#lerna-changelog)
+
+#### How to use
+Pass your github token to generate the changelog automatically.
+In your terminal, execute the following command:
+```bash
+GITHUB_AUTH=xxxxx lerna-changelog
```

Comment:
Why does this require a GitHub token? I would have expected it to be able to get the commit info simply from the git history, or is it pulling pull request info as well?

Could we at least ask people to use a read-only token?

Reply:
>   Why does this require a GitHub token? 

For rate limiting issues. Sadly, it's required...

> or is it pulling pull request info as well?

It will use pull request info

Also, it will print changelog for PR **that have been labelled** (see lerna.json)

> Could we at least ask people to use a read-only token?

Yes. Good idea.